main.py

```diff
@@ -1,70 +1,92 @@
-"""FAST API DEMO"""
+"""FAST API DEMOS"""
 from datetime import datetime
 import pyodbc
 from fastapi import FastAPI, HTTPException
-from pydantic import BaseModel
+from pydantic import BaseModel, validator
 
 
 app = FastAPI()
-
-# Replace the server, database, username, and password with your own values
-# connection_string = 'Driver={ODBC Driver 17 for SQL Server};Server=localhost
-# ,1433;Database=Sales;UID=sa;PWD=Password123;'
-connection_string = "Driver={ODBC Driver 17 for SQL Server};\
-Server=localhost,1433;UID=sa;PWD=Password123;"
-
-cnxn = pyodbc.connect(connection_string, autocommit=True)
+CONNECTION_STRING = "Driver={ODBC Driver 17 for SQL Server};\
+                    Server=localhost,1433;UID=sa;PWD=Password123;"
+DB_NAME = "Vegards_sales"
+cnxn = pyodbc.connect(CONNECTION_STRING, autocommit=True)
 
 
 class Sale(BaseModel):
     """Class for a sale"""
 
     sales_date: str
     buyer: str
-    apples: int
-    oranges: int
+    apples: str
+    oranges: str
```

Comment:
These two need to be optional with default 0. Because you can buy only oranges or apples.